### DISCUSSION TITLE
Newer Hue Motion Sensor without OTA-Support

### DISCUSSION THREAD

**Discussion by GerdRuetten** [Koenkk/zigbee2mqtt#24940] [Devices]
Dear all,

I have previously used the older model of the Hue motion detector, model 9290012607 (https://www.zigbee2mqtt.io/devices/9290012607.html). This model supports OTA updates.

Now I have bought the newer model 9290030675 (https://www.zigbee2mqtt.io/devices/9290030675.html).

But the model does not support OTA.

It is neither displayed in the OTA register in z2m, nor are the update entities (update_available, update_state and update button) passed to HomeAssistant.

Is the pairing faulty or does the newer model actually not support OTA?

Thank you in advance for helpful answers!

```
Zigbee2MQTT version
1.41.0-dev commit: [267d800]
Coordinator type
EmberZNet
Coordinator revision
7.4.2 [GA]
Coordinator IEEE Address
0x4c5bb3fffe9307c8
Frontend version
0.7.5
zigbee-herdsman-converters version
20.54.0
zigbee-herdsman version
2.1.9
```

Best regards
Gerd

**Comment by mundschenk-at** [comment]
From what I understand there is currently no newer firmware for these devices.

**Comment by Grandma-Betty** [comment]
I mentioned the same issue. I got four Philips Hue motion sensors based on model [9290012607](https://www.zigbee2mqtt.io/devices/9290012607.html#philips-9290012607) which actually support OTA in Z2M and two Philips Hue motion sensors based on model [9290030675](https://www.zigbee2mqtt.io/devices/9290030675.html#philips-9290030675) which do not support OTA in Z2M.
@mundschenk-at It may be right what you wrote that there is no firmware update available, but even if there is no firmware update, the according firmware version data fields in the "update" data section should still be exposed if supported by the device I guess? Please correct me if I'm wrong.

Here is the "update" data section of the model **9290012607**:
![3](https://github.com/user-attachments/assets/db6ed83c-6bce-42a3-862d-cc14476cd40f)

On the **9290030675** model the according firmware version data fields are missing on the "update" section:
![4](https://github.com/user-attachments/assets/bb813400-e2ca-42cf-a413-2a67b1ad8f64)

Thank you in advance.

**Reply by mundschenk-at** [reply]
Yes, but no. AFAIK for (most?) devices that technically support FW updates (i.e. have the relevant ZCL cluster), but have not actually got a FW update in https://github.com/Koenkk/zigbee-OTA, the converter does not expose the update capability. I am surprised that there's an `update` key in your state object at all. Mine don't show that. Are you on the `1.x` branch or already on `2.0`?

**Reply by Grandma-Betty** [reply]
I am already on Z2M `2.0.0` but honestly I cannot tell anymore if there was a difference to `1.42.x` according to missing version data fields because I mentioned just today about that issue of differences between those two device models. I am not 100% sure if those two devices were showing up in the Z2M OTA menu on version `1.42.x` or not.

**Reply by mundschenk-at** [reply]
The new ones should not show up. Strangely, I'm on `2.0.0` (or really `dev`, but currently same as `2.0.0`) as well. For mine, the `update` key simply does not exist in the state object.

**Reply by Grandma-Betty** [reply]
Well, I deleted and re-paired one of the two according devices and now the `update` key has disappeared. Very strange though.

**Comment by mundschenk-at** [comment]
There is now an updated firmware for the Hue Motion Sensor (2021) and the latest release should show the update entity.
